src/Core/ServiceWrapper/Main.cs

```diff
@@ -589,6 +591,21 @@ public static void Run(string[] _args, ServiceDescriptor? descriptor = null)
                 args = args.GetRange(2, args.Count - 2);
             }
 
+            bool elevated;
+            if (args[0] == "/elevated")

```

Comment:
Maybe it still makes sense to document it as internal argument. It can be used by advanced users who know what they are doing and want to improve performance a bit. No strong opinion

Reply:
It makes more sense to add a `-no-elevate` argument. I'll add it as part of #433 if there is no priority.

Reply:
Agreed